### PR TITLE
Fixes #8652: replace errata content search links

### PR DESCRIPTION
**Pull request by waldenraines**
Replace errata content search links with links to the errata page
filtered by the correct repository.

http://projects.theforeman.org/issues/8652

### CODE REVIEW

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/filters/views/filter-repositories.html [hunk 1]
```diff
@@ -115,7 +115,7 @@
                 </div>
 
                 <div>
-                  <a translate ng-href="/katello/content_search#/!=&search[subgrid][repo_id]={{ repository.id }}&search[subgrid][type]=repo_errata">
+                  <a ng-href="/errata?repositoryId={{ repository.id }}" translate>
```
Comment (ehelms) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/filters/views/filter-repositories.html]: Could we use `ui-sref` instead?

Comment (waldenraines) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/filters/views/filter-repositories.html]: Yeah, good call, will update.

**Comment by waldenraines** [comment]
@ehelms updated.

Note that this PR depends on https://github.com/Katello/bastion/pull/35 to function properly.

**Comment by ehelms** [comment]
ACK

**Pull request merged**